executor/infoschema_reader.go

```diff
@@ -2011,6 +2014,18 @@ func (e *memtableRetriever) setDataForClientErrorsSummary(ctx sessionctx.Context
 	return nil
 }
 
+func (e *memtableRetriever) setDataForTiDBTrx(ctx sessionctx.Context) error {
+	sm := ctx.GetSessionManager()
+	if sm == nil {
+		return nil
+	}
+	infoList := sm.ShowTxnList()
+	for _, info := range infoList {
+		e.rows = append(e.rows, info.ToDatum())
+	}
+	return nil
+}
```

Comment:
Looks like here needs some permission check. The txn list should not be visible to everyone.